components/brave_wallet_ui/page/screens/onboarding/import-or-restore-wallet/import-or-restore-wallet.tsx

```diff
@@ -75,15 +74,6 @@ export const OnboardingImportOrRestoreWallet = () => {
             </CardButton>
           )}
 
```

Comment:
Please remove this alias braveWalletImportFromLegacy -> IDS_BRAVE_WALLET_IMPORT_FROM_LEGACY